server/src/main/java/app/friendship/FriendshipService.java

```diff
@@ -51,41 +52,73 @@ public void acceptFriendRequest(Integer friend1Id, Integer friend2Id) {
 
     public void removeFriend(Integer senderId, Integer friendId) {
         friendshipRepository.deleteAll(friendshipRepository.findByStatusAndSenderId(
-            FriendshipStatus.ACCEPTED,
-            senderId
+                FriendshipStatus.ACCEPTED,
+                senderId
         ));
 
         friendshipRepository.deleteAll(friendshipRepository.findByStatusAndSenderId(
-            FriendshipStatus.ACCEPTED,
-            friendId
+                FriendshipStatus.ACCEPTED,
+                friendId
         ));
     }

```

Comment:
by this logic you will remove all friends of both guys : D gj . i will fix it in my commit